src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -83,10 +83,12 @@ private Runtime() {}
      * until they finish.  Once this is done the virtual machine
      * {@linkplain #halt halts}.
      *
-     * <p> If this method is invoked from more than one thread the status code
-     * of the first invocation will be used; the status codes from other invocations
-     * will be ignored. If this method is invoked from a shutdown hook the system
-     * will deadlock.
+     * <p> Shutdown is serialized such that only one invocation will run
+     * shutdown hooks and terminate the VM with the given status code. That
+     * invocation may be initiated via platform specific signal handlers. All
+     * other invocations will block indefinitely, and their supplied exit
+     * statuses will be ignored. If this method is invoked from a shutdown hook
```

Comment:
If they block indefinitely then it is implicit that nothing happens with their exit status. I think you are trying to say too much.